webui/apps/docs/src/content/docs/docs/howto/use_k8s.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploy Eidolon AI with Kubernetes
+description: Learn how to deploy and manage Eidolon AI in a Kubernetes environment
+---
+
+Running Eidolon AI on Kubernetes unlocks powerful deployment and scaling capabilities. This guide walks you through setting up Eidolon AI in your Kubernetes cluster.
+
+## Prerequisites
+
+- A running Kubernetes cluster and `kubectl` CLI ([Get started with Minikube][minikube])
+- Basic familiarity with [Eidolon AI](https://www.eidolonai.com/docs/)
+
+## Quick Setup
+
+1. Clone the quickstart repo:
+   ```bash
+   git clone git@github.com:eidolon-ai/eidolon-quickstart.git
+   ```
+
+2. Create a namespace (optional for Make users):
+   ```bash
+   kubectl create namespace eidolon-quickstart
+   ```
+
+3. Set up API access tokens using Kubernetes secrets:
+   ```bash
+   # Option 1: From .env file
+   kubectl create secret generic eidolon --from-env-file=.env
+
+   # Option 2: Direct configuration
+   kubectl create secret generic eidolon \
+     --from-literal=OLLAMA_URL=https://ollama.example.com \
+     --from-literal=OPENAI_API_KEY=your-openai-api-key \
+     --from-literal=MISTRAL_API_KEY=your-mistral-api-key \
+     --from-literal=ANTHROPIC_API_KEY=your-anthropic-api-key
+   ```
+   Add `-n eidolon-quickstart` if using a custom namespace.
+
+## Eidolon Kubernetes Operators
+
+Eidolon uses three Kubernetes operators to manage its components:
+
+- **Machine Operator**: Manages core Eidolon infrastructure
+- **Agent Operator**: Handles AI agent lifecycle
+- **Reference Operator**: Controls resource references
+
+To use your secrets in deployments, add this to your deployment YAML:
+```yaml
+envFrom:
+  - secretRef:
+      name: eidolon
+```
+
+## Installation
+
+Choose **either** the Make-based or Helm-based installation method:
+
+### Option 1: Using Make (Simplified Setup)
+
+Make handles everything in the default namespace and is great for getting started quickly:
+
+```bash
+# Install operator and dependencies
+make k8s-operator
+
+# Build and deploy your application
+make docker-build
+make k8s-serve
+```
+
+### Option 2: Using Helm (Advanced Setup)
+
+Helm installation gives you more control over namespaces and configuration:
+
+1. Install the Eidolon operator:
+   ```bash
+   helm repo add eidolon-operator https://eidolonai.com/charts 
+   helm install eidolon eidolon-operator/eidolon-operator-chart \
+     --namespace eidolon-operator-system --create-namespace
```

Comment:
What do you think about removing the --namespace flag here? It would be good to copy the helm literal commands as much as possible. If somebody wants to define resources in a custom namespace they will already know the flag.

Alternatively, passing in the namespace in the makefile command might be good. I worry about the two paths creating different environments.